frontend/src/pages/TermPlanner/TermBox/TermBox.jsx

```diff
@@ -35,11 +34,6 @@ const TermBox = ({
 
   const isSmall = useMediaQuery("(max-width: 1400px)");
 
-  const iconStyle = {
```

Comment:
can use the `useTheme` hook here rather than styling the icon components. i.e.

```
import { useTheme } from "styled-components";
...
const theme = useTheme();
...
const iconStyle = {
  fontSize: "12px",
  color: theme.termCheckbox.color
}
```